pkgs/applications/misc/fuzzel/default.nix

```diff
@@ -1,22 +1,76 @@
-{ stdenv, lib, fetchzip, pkg-config, meson, ninja, wayland, pixman, cairo, librsvg, wayland-protocols, wlroots, libxkbcommon, scdoc, git, tllist, fcft}:
+{ stdenv
+, lib
+, fetchFromGitea
+, pkg-config
+, meson
+, ninja
+, wayland-scanner
+, wayland
+, pixman
+, wayland-protocols
+, libxkbcommon
+, scdoc
+, tllist
+, fcft
+, enableCairo ? true
+, enablePNG ? true
+, enableSVG ? true
+# Optional dependencies
+, cairo
+, librsvg
+, libpng
+}:
+
+let
+  # Courtesy of sternenseemann and FRidh, commit c9a7fdfcfb420be8e0179214d0d91a34f5974c54
+  mesonFeatureFlag = opt: b:
+    "-D${opt}=${if b then "enabled" else "disabled"}";
+in
 
 stdenv.mkDerivation rec {
   pname = "fuzzel";
   version = "1.6.1";
 
-  src = fetchzip {
-    url = "https://codeberg.org/dnkl/fuzzel/archive/${version}.tar.gz";
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "dnkl";
+    repo = "fuzzel";
+    rev = version;
     sha256 = "sha256-JW5sAlTprSRIdFbmSaUreGtNccERgQMGEW+WCSscYQk=";
```

Comment:
Did you verify this sum?

Reply:
yes, I did verify the checksum, because fetchFromGitHub be default disable fetching submodules, I believe fetchzip and fetchFromGitea perform the same download.